app/services/bufferizer/facebook_post.rb

```diff
@@ -0,0 +1,37 @@
+module Bufferizer
+  class FacebookPost
+    def self.call(article, post, admin_id = nil)
```

Comment:
Is `admin_id` really optional? All actual calls in `app/controllers/admin/buffer_updates_controller.rb` seem to pass it in, only the specs don't.

Reply:
Good catch, updated!